src/core/connection.c

```diff
@@ -485,6 +485,32 @@ QuicConnUninitialize(
         QuicBindingRemoveConnection(Connection->Paths[0].Binding, Connection);
     }
 
+    if (Connection->State.EncryptionOffloading) {
+        CXPLAT_QEO_CONNECTION Offloads[] = {
```

Comment:
You will have to fill out the connection IDs at least.

Reply:
Which should be used? OrigDestCID? or which one of DestCIDs

Reply:
The destination IP and CID should be used, where "destination" depends on the direction. For TX, it is the peer's, but for RX, it's our local one.